CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
This repo already has at least one version published. Should it be listed here? What about a patch version bump for the inclusion of the changes in this PR?

Reply:
Ah yep, was planning to file a separate PR which details the prior versions released, as well as the changelog for the impending v0.0.3 release.